prime_directive/core/scribe.py

```
@@ -1,20 +1,44 @@
+        text (str): The input text to tokenize.
+        model (str): The model name used to select the encoding; if model-specific encoding cannot be obtained, a base encoding is used.
+
+    Returns:
+        int: The token count for the input text according to the chosen encoder, or 0 if tokenization fails.
+    """
+    try:
+        import tiktoken
+
+        try:
+            enc = tiktoken.encoding_for_model(model)
+        except Exception:
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)